ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -124,385 +129,392 @@ Control {
-                Layout.fillHeight: true
-                Layout.alignment: Qt.AlignCenter
-                visible: collectiblesView.count == 0
+                anchors.centerIn: parent
+                visible: (collectiblesView.count === 0)
                 horizontalAlignment: Text.AlignHCenter
                 verticalAlignment: Text.AlignVCenter
                 color: Theme.palette.directColor1
                 text: qsTr("%1 has not shared any collectibles").arg(root.mainDisplayName)
             }
-            Item {
-                Layout.fillWidth: true
-                Layout.fillHeight: true
-                Layout.topMargin: (collectiblesView.contentY > Style.current.padding) ? 1 : Style.current.padding
-                Behavior on Layout.topMargin { NumberAnimation { duration: 50 } }
-                clip: true
-                StatusGridView {
-                    id: collectiblesView
-                    width: 608
-                    height: parent.height
-                    anchors.top: parent.top
-                    anchors.topMargin: Style.current.halfPadding
-                    anchors.bottom: parent.bottom
-                    anchors.bottomMargin: Style.current.halfPadding
-                    anchors.horizontalCenter: parent.horizontalCenter
-                    anchors.horizontalCenterOffset: Style.current.halfPadding
-                    cellWidth: d.delegateWidthS
-                    cellHeight: d.delegateHeightS
-                    visible: count
-                    clip: false
-                    // TODO Issue #11637: Dedicated controller for user's list of collectibles (no watch-only entries)
-                    model: collectiblesProxyModel
-                    ScrollBar.vertical: StatusScrollBar { }
-                    delegate: StatusRoundedImage {
-                        width: GridView.view.cellWidth - Style.current.padding
-                        height: GridView.view.cellHeight - Style.current.padding
-                        border.width: 1
-                        border.color: Theme.palette.directColor7
+            StatusGridView {
+                id: collectiblesView
+                width: 608
+                height: parent.height
+                anchors.top: parent.top
+                anchors.topMargin: (webView.contentY > Style.current.halfPadding) ? 1 : Style.current.bigPadding
+                Behavior on anchors.topMargin { NumberAnimation { duration: 50 } }
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: Style.current.halfPadding
+                anchors.horizontalCenter: parent.horizontalCenter
+                anchors.horizontalCenterOffset: Style.current.halfPadding
+                cellWidth: d.delegateWidthS
+                cellHeight: d.delegateHeightS
+                visible: count
+                clip: false
+                // TODO Issue #11637: Dedicated controller for user's list of collectibles (no watch-only entries)
```

Comment:
Can you explain that?? I don't get it

Reply:
hmm, I think we need to remove this. Doesn't make sense to me either. Probably was an older comment